servers/dsub/jobs/controllers/capabilities_controller.py

```diff
@@ -1,7 +1,43 @@
+from jobs.controllers.utils.providers import ProviderType
+from jobs.models.authentication_capability import AuthenticationCapability
+from jobs.models.capabilities_response import CapabilitiesResponse
+
+
 def get_capabilities():
     """Get the capabilities of this backend implementation.
 
     Returns:
         CapabilitiesResponse: Response containing this backend's capabilities
     """
-    raise NotImplemented('Not yet supported.')
+    capabilities = CapabilitiesResponse(
+        authentication=AuthenticationCapability(
+            is_required=true,
+            scopes=[
+                'https://www.googleapis.com/auth/genomics',
+                'https://www.googleapis.com/auth/cloudplatformprojects.readonly'
+            ]),
+        display_fields={
+            'name': 'Job',
+            'status': 'Status',
+            'submission': 'Submitted',
+            'labels.job_id': 'Job ID',
+            'labels.task_id': 'Task ID',
+            'extensions.user_id': 'User ID',
+            'extensions.status_detail': 'Status Detail'
+        },
+        common_labels=['job_id', 'task_id'],
+        extended_query_fields=['project_id', 'user_id', 'submission'])
+
+    if _provider_type() == ProviderType.GOOGLE:
+        capabilities.authentication = AuthenticationCapability(
```

Comment:
Did you mean to leave this unset above?

Reply:
Ah.. beat me to it.

Reply:
Yup sorry ... noticed that right as I sent it for review